### PR TITLE
Update release post maven and gradle instructions

### PR DESCRIPTION
**Pull request by mbroz2**

### CODE REVIEW

**Review by scottkurz — APPROVED** [review]

File: templates/beta-release-post.adoc [hunk 1]
```diff
@@ -117,7 +117,7 @@ If you're using link:{url-prefix}/guides/maven-intro.html[Maven], you can instal
 <plugin>
     <groupId>io.openliberty.tools</groupId>
     <artifactId>liberty-maven-plugin</artifactId>
-    <version>RELEASE</version>
+    <version>3.8.2</version>
```
Comment (cherylking) [inline templates/beta-release-post.adoc:120]: Does `RELEASE_VERSION` in the `runtimeArtifact` get replaced by the actual version when you build the blog post?

File: templates/ga-release-post.adoc [hunk 2]
```diff
@@ -115,26 +115,36 @@ Check out link:{url-prefix}/blog/?search=release&search!=beta[previous Open Libe
...
 ----
-dependencies {
-    libertyRuntime group: 'io.openliberty', name: 'openliberty-runtime', version: '[RELEASE_VERSION,)'
+buildscript {
+    repositories {
+        mavenCentral()
+    }
+    dependencies {
+        classpath 'io.openliberty.tools:liberty-gradle-plugin:3.6.2'
+    }
```
Comment (cherylking) [inline templates/ga-release-post.adoc:139]: What about the portion that applies the Liberty Gradle plugin? That is also necessary.

`apply plugin: 'liberty'`

File: templates/beta-release-post.adoc [hunk 3]
```diff
@@ -151,10 +151,24 @@ Or for link:{url-prefix}/guides/gradle-intro.html[Gradle]:
...
 [source,gradle]
 ----
+buildscript {
+    repositories {
+        mavenCentral()
+    }
+    dependencies {
+        classpath 'io.openliberty.tools:liberty-gradle-plugin:3.6.2'
+    }
+}
```
Comment (cherylking) [inline templates/beta-release-post.adoc:161]: What about the portion that applies the Liberty Gradle plugin? That is needed also.

`apply plugin: 'liberty'`

**Review by cherylking — CHANGES_REQUESTED** [review]

File: templates/beta-release-post.adoc (hunk 1 above)
Comment (mbroz2) [reply templates/beta-release-post.adoc:120]: yes; it gets replaced with the Liberty version for which the blog post is being created (e.g. 23.0.0.10 or 23.0.0.10-beta)

File: templates/beta-release-post.adoc (hunk 3 above)
Comment (mbroz2) [reply templates/beta-release-post.adoc:161]: How should this be added in the scope of the above?  Could you use the 'Add a suggestion' to add it how you believe it should be presented to the end user?

File: templates/ga-release-post.adoc (hunk 2 above)
Comment (mbroz2) [reply templates/ga-release-post.adoc:139]: How should this be added in the scope of the above?  Could you use the 'Add a suggestion' to add it how you believe it should be presented to the end user?

File: templates/beta-release-post.adoc (hunk 3 above)
Comment (cherylking) [inline templates/beta-release-post.adoc:161]:
```suggestion
}
apply plugin: 'liberty'
```

File: templates/ga-release-post.adoc [hunk 4]
```diff
@@ -115,26 +115,36 @@ Check out link:{url-prefix}/blog/?search=release&search!=beta[previous Open Libe
...
 }
```
Comment (cherylking) [inline templates/ga-release-post.adoc:140]:
```suggestion
}
apply plugin: 'liberty'
```

**Review by cherylking — APPROVED** [review]

**Pull request merged**